mirage/config.js

```diff
@@ -464,19 +464,19 @@ export default function() {
   // DELETE /user-categories/:id
   this.delete('/user-categories/:id');
 
-  /////////////
+  // ///////////
   // User roles
-  /////////////
+  // ///////////
 
   // POST /user-roles
   this.post('/user-roles');
 
   // DELETE /user-roles
   this.delete('/user-roles/:id');
 
-  //////////////
+  // ////////////
   // User skills
-  //////////////
+  // ////////////
```

Comment:
These are rather awkward changes. Should we be doing this another way altogether?